tools/db_bench_tool.cc

```diff
@@ -7340,6 +7351,42 @@ int db_bench_tool(int argc, char** argv) {
     FLAGS_env = new ROCKSDB_NAMESPACE::HdfsEnv(FLAGS_hdfs);
   }
 
+#ifndef ROCKSDB_LITE
+  // Check for the encryption flag (and create an encrypted environment)
+  Random rnd(301);
+  std::shared_ptr<EncryptionProvider> provider;
+  if (FLAGS_encrypt_db && !strcasecmp(FLAGS_block_cipher.c_str(), "IPP_AES")) {
+    std::string key(FLAGS_block_cipher_key_size_in_bytes, ' ');
+    for (int i = 0; i < FLAGS_block_cipher_key_size_in_bytes; ++i)
+      key[i] = rnd.Uniform(256) & 0xFF;
```

Comment:
By using a random key like this, can I still run db_bench multiple times over the same data set successfully?  If so, is there a reason not to make the random key the "test" key and just use TEST_Initialize?

Reply:
Same. Will add the key as a flag and remove this.